readme.md

```diff
@@ -4,8 +4,8 @@
     * Prefer descriptive names over short form
     * Follow Coding Convention
     * The class name should be a noun
-    * Avoid clutters like _l, m_, o_,O1, Ol, lO, etc
-    * Avoid using non-ASCII characters and words from the local language
+    * Avoid ambiguious names like _l, m_, o_,O1, Ol, lO, etc
```

Comment:
i think clutter name can work fine, is there any benefit of doing it.

Reply:
nope clutter doesn't convey your actual meaning of using ambiguous characters.
the spelling of ambiguous is wrong.